src/Chainweb/Transaction.hs

```diff
@@ -42,6 +38,9 @@ data PayloadWithText = PayloadWithText
     deriving (Show, Eq, Generic)
     deriving anyclass (NFData)
 
+instance Ord PayloadWithText where
```

Comment:
As I said over in the other spot, `Hashable` is much better here -- rather than ord-comparing payload bytes (yuck), there is a hash code inside `Payload` that you can just project out and use when implementing `hashWithSalt`. Come to think of it, the derived `Eq` instance is also pretty bad here, we should just make this an `==` on `payloadBytes` because otherwise `payloadObj` is going to be compared for equality also.